### PR TITLE
Tweak app's OS menu (populate help section)

### LINKED ISSUE #37: Improve the help menu
The help menu is empty :disappointed: (see screenshot below)
It would be nice to have:
1. The wallet version
2. A "Report a bug" link which points to a new issue on this repo (i.e. https://github.com/alephium/alephium-wallet/issues/new) to let users easily provide feedback
3. Optionally a help/support link which points either to some documentation or to the discord chat

Also the "File" and especially the "Window" menus feel a bit useless and could be hidden.

2021-10-30-033016_1916x1056_scrot

> Note: This is inspired by the MyCrypto desktop wallet help menu

### PR DESCRIPTION
**Pull request by mvaivre**
Fixes #37 

Redirect to issue creation (github) to allow reporting a bug + link to discord server for seeking help.

### CODE REVIEW

File: public/electron.js [hunk 1]
```diff
@@ -15,18 +15,108 @@
...
+      {
+        label: 'Report a bug',
+        click: async () => {
+          await shell.openExternal('https://github.com/alephium/alephium-wallet/issues/new')
+        }
+      },
+      {
+        label: 'Get some help',
+        click: async () => {
+          await shell.openExternal('https://discord.gg/nD8FzcTugn')
```
Comment (polarker) [inline public/electron.js:110]: Just to double check: is this a permanent discord invitation link ?

File: public/electron.js [hunk 2]
```diff
@@ -15,18 +15,108 @@
...
+    submenu: [
+      { role: 'minimize' },
+      ...(isMac ? [{ role: 'zoom' }, { type: 'separator' }, { role: 'front' }] : [{ role: 'close' }])
+    ]
+  },
+  {
+    role: 'help',
+    submenu: [
+      ...(isMac ? [] : [{ role: 'about' }]),
+      { type: 'separator' },
```
Comment (polarker) [inline public/electron.js:100]: We need to put this `separator` only for non-Mac

File: public/electron.js [hunk 3]
```diff
@@ -15,18 +15,108 @@
...
+      {
+        label: 'Report a bug',
```
Comment (polarker) [inline public/electron.js:102]: Let's use `Report an issue` to be consistent with Github issues

File: public/electron.js (hunk 1 above)
Comment (mvaivre) [reply public/electron.js:110]: I believe so. It's the link we've been using on our website.

File: public/electron.js (hunk 2 above)
Comment (mvaivre) [reply public/electron.js:100]: It's not reflected in the UI, but can indeed be removed.

File: public/electron.js (hunk 3 above)
Comment (mvaivre) [reply public/electron.js:102]: Alright

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**